src_ext/Makefile

```diff
@@ -33,12 +33,16 @@ MD5_PKG_$(1) = $(MD5_$(1))
 endef
 
 SRC_EXTS = cppo extlib re cmdliner ocamlgraph cudf dose3 opam-file-format result seq stdlib-shims
-PKG_EXTS = $(SRC_EXTS) dune-local findlib ocamlbuild topkg mccs
+PKG_EXTS = $(SRC_EXTS) dune-local findlib ocamlbuild topkg mccs opam-0install-cudf 0install-solver
```

Comment:
FWIW, this isn't needed now - it'll pick it up through the SRC_EXTS below

Reply:
fixed! TIL Make variable are lazy ^^